Core/Src/main.c

```diff
@@ -95,19 +96,61 @@ int main(void)
   MX_ADC_Init();
   MX_TIM16_Init();
   MX_USART1_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
 
   /* USER CODE BEGIN 2 */
-  debug("\n\nBootcamp starting up...");
-  debug("Compiled on %s at %s", __DATE__, __TIME__);
 
+  //start pwm
+  HAL_TIM_PWM_Start(&htim1, TIM_CHANNEL_1);
```

Comment:
Generally good practice to create all variables and such before calling functions, but this will still function.